baseapp/baseapp_test.go

```diff
@@ -236,6 +236,7 @@ func TestSetLoader(t *testing.T) {
 	v := []byte("value")
 
 	for name, tc := range cases {
+		tc := tc
```

Comment:
I don't like this tbh

Reply:
https://github.com/kyoh86/scopelint

Reply:
Interesting i didn’t know this was an issue